config/jest/default.js

```diff
@@ -1,5 +1,5 @@
 module.exports = {
-  collectCoverage: true,
+  // collectCoverage: true,
```

Comment:
Not sure, why have you commented this? It is not related to the API key.

Reply:
This wasn't supposed to be in the PR, I'll clean this up.